webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/EntityJoinRelation.java

```diff
@@ -7,9 +7,9 @@
  */
 package io.camunda.webapps.schema.entities.usermanagement;
 
-public record EntityJoinRelation<T>(String name, T parent) {
+public record EntityJoinRelation(String name, String parent) {
```

Comment:
ℹ️ I de-generified this class as I noticed we always use String-based IDs and I don't think that will change in the future. Furthermore, the class is internal, so we can refactor the previous behavior in the future without much impact to users.